rdf-toolkit.md

```diff
@@ -0,0 +1,18 @@
+# How to start:
+  - on windows: drag the ontology file to the "rdf-toolkit.bat" file
+    or start the `cmd.exe` in the current directory
+    run the `rdf-toolkit.bat` script with the ontology file as an argument, e.g.:
+    ```
+    rdf-toolkit.bat ontology.ttl
+    ```
+  - on Unix-like systems: run the `rdf-toolkit.sh` script with the ontology file as an argument, e.g.:
+    ```bash
+    ./rdf-toolkit.sh ontology.ttl
```

Comment:
please correct the current ontology name in our repository
```suggestion
    ./rdf-toolkit.sh ontohgis.ttl
```

Reply:
Resolved in commit 49fff4a